packages/blade/src/tokens/global/motion.ts

```diff
@@ -0,0 +1,132 @@
+import makeBezier from '../../utils/makeBezier';
+
+export type Motion = Readonly<{
+  delay: {
+    /** `70` milliseconds */
+    '2xshort': number;
+    /** `120` milliseconds */
+    xshort: number;
+    /** `180` milliseconds */
+    short: number;
+    /** `3000` milliseconds */
+    long: number;
+    /** `5000` milliseconds */
+    xlong: number;
+  };
+  duration: {
+    /** `70` milliseconds */
+    quick1: number;
+    /** `150` milliseconds */
+    quick2: number;
+    /** `200` milliseconds */
+    quick3: number;
+    /** `250` milliseconds */
+    moderate1: number;
+    /** `300` milliseconds */
+    moderate2: number;
+    /** `400` milliseconds */
+    gentle1: number;
+    /** `600` milliseconds */
+    gentle2: number;
+    /** `900` milliseconds */
+    gentle3: number;
+  };
+  easing: {
+    /** Easings for all standard animations*/
+    standard: {
+      /** `cubic-bezier(0.5, 0, 0.3, 1.5)`
+       *
+       *  Returns a `string` for web & react-native-reanimated's Easing Function of type `EasingFunctionFactory` for native */
+      attentive: string;
```

Comment:
Not sure what we can do for `Easing.Bezier` since the return type is always `{ factory: () => (value: number) => number };`. It stores the bezier values internally. It's sorta like this `Easing.Bezier(1,0,1,0)(0)`.